doc/cephfs/fs-volumes.rst

```diff
@@ -585,6 +589,33 @@ Use a command of the following form to remove the earmark of a subvolume:
 
    ceph fs subvolume earmark rm <vol_name> <subvol_name> [--group_name <subvol_group_name>]
 
+Getting enctag of a subvolume
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use a command of the following form to get the enctag on a subvolume:
+
+.. prompt:: bash #
+
+   ceph fs subvolume enctag get <vol_name> <subvol_name> [--group_name <subvol_group_name>]
+
```

Comment:
This needs more explanation for what an enctag is, how it will be used in practice, etc.

One might also wonder, why not use `ceph fs subvolume metadata set`?

Reply:
>This needs more explanation for what an enctag is, how it will be used in practice, etc.
Yes I'll add more info about it.

>One might also wonder, why not use ceph fs subvolume metadata set
I considered that and chose this way. Did not feel strongly about either way, but slightly this way.